src/types/api.ts

```diff
@@ -158,6 +158,12 @@ export type UserInfo = {
   display_name: string | null;
 };
 
+export type Mention = UserInfo & {
+  userid: string;
+  username: string;
+  link: string;
```

Comment:
What does `link` point to? The user's profile page? What about LMS users in future who don't have profile pages?